src/components/NcAppNavigationToggle/NcAppNavigationToggle.vue

```diff
@@ -1,15 +1,42 @@
 <!--
   - SPDX-FileCopyrightText: 2019 Nextcloud GmbH and Nextcloud contributors
   - SPDX-License-Identifier: AGPL-3.0-or-later
+  -
+  - This component is only used for the NcAppNavigation component and not exported otherwise.
```

Comment:
Let's separate SPDX comment and component's documentation.

I'm afraid it may cause issues on our SPDX scripts, or if we add ESLint plugin for SPDX.